unstructured/partition/xlsx.py

```diff
@@ -59,7 +62,7 @@ def partition_xlsx(
     page_number = 0
     for sheet_name, table in sheets.items():
         page_number += 1
-        html_text = table.to_html(index=False, header=False, na_rep="")
+        html_text = table.to_html(index=False, header=include_header, na_rep="")
```

Comment:
Couldn't find the`.to_html` method at Table, Text or Element, where is the code for that?

Reply:
This is the definition VSC points me to:
/home/coni/.pyenv/versions/3.11.0/envs/unstructured11/lib/python3.11/site-packages/pandas/core/frame.py
Should be this:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_html.html